src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -4,45 +4,52 @@
 
 import java.util.List;
 
-import seedu.address.commons.core.index.Index;
 import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.logic.Messages;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
+import seedu.address.model.person.Id;
 import seedu.address.model.person.Person;
 
+
 /**
  * Deletes a person identified using it's displayed index from the address book.
  */
 public class DeleteCommand extends Command {
 
-    public static final String COMMAND_WORD = "delete";
+    public static final String COMMAND_WORD = "/remove";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + ": Removes the person identified by the id number provided.\n"
+            + "Parameters: ID (must be a positive 6-digit number)\n"
+            + "Example: " + COMMAND_WORD + " 240001";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Removed Person: %1$s";
 
-    private final Index targetIndex;
+    private final Id ID;
```

Comment:
Perhaps can consider changing the variable name from "ID" to "id" since we consistently use "id" throughout all the files?